components/learningUnits/LearningUnitInfo.js

```diff
@@ -15,9 +16,12 @@ export default function LearningUnitInformation({ learningUnitId }) {
     return <Alert severity="error">Error</Alert>;
   }
 
+  const goBackPageText = "Back to curriculum";
```

Comment:
I think it's unnecessary to define the constant before, it's very clear if you just give the text as the prop to the component

Reply:
Done!